src/components/WebcamCapture/WebcamCaptureContainer.js

```diff
@@ -57,6 +63,7 @@ class WebcamCaptureContainer extends Component {
 	render() {
 		return (
 			<WebcamCapture
+				isDetecting={this.state.isDetecting}
```

Comment:
does this need to be passed as a prop? or should the visibility of the webcamCapture be controlled in its container?